src/components/dashboard/commonConsentSections/FirstCommonConsentSection.tsx

```diff
@@ -152,6 +156,11 @@ function FirstCommonConsentSection({
                     'This study will only require you to answer weekly survey questions.',
                   )
                   setErrorMessage('')
+                  updateClientData(
+                    step,
+                    'howToParticipate',
```

Comment:
Could we add a constants file, so we centralize all keys - values if we need to adjust in the near future.

Reply:
indeed